### PR TITLE
Remove ActAsUserPresenter

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: none
affects: none
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/luysozp) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fluysozp)
teacher: [Link](https://rebrand.ly/c2943xl) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fc2943xl)
parent: [Link](https://rebrand.ly/xjjuwo0) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fxjjuwo0)

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: None

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.58% | 93.6% | -0.03%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/ActAsUser/ActAsUserViewController.swift [hunk 1]
```diff
@@ -18,121 +18,142 @@
...
+        let animation = CAKeyframeAnimation(keyPath: "transform")
+        let easeInEaseOut = CAMediaTimingFunction(name: .easeInEaseOut)
+        let cover = CATransform3DIdentity
+        let aside = CATransform3DMakeAffineTransform(redPanda.transform.translatedBy(x: -82, y: 30).rotated(by: .pi / -6))
+        animation.timingFunctions = [ easeInEaseOut, easeInEaseOut, easeInEaseOut, easeInEaseOut ]
+        animation.duration = 10
+        animation.keyTimes = [ 0, 0.2, 0.35, 0.85, 1 ]
+        animation.values = [ aside, aside, cover, cover, aside ]
+        animation.repeatCount = .infinity
+        redPanda.layer.add(animation, forKey: "masquerade")
```
Comment (armstrongnate) [inline Core/Core/ActAsUser/ActAsUserViewController.swift]: Wow. Much nicer!

File: Core/Core/ActAsUser/ActAsUserViewController.swift [hunk 2]
```diff
@@ -18,121 +18,142 @@
...
-                        }
-                        redPanda.transform = redPanda.transform
-                            .rotated(by: .pi / -6)
-                        self?.view.layoutIfNeeded()
-                    }, completion: { [weak self] _ in
-                        self?.animatePanda()
-                    })
-            })
-        }
+    @objc func keyboardWillChangeFrame(_ notification: Notification) {
```
Comment (vanwagonet) [inline Core/Core/ActAsUser/ActAsUserViewController.swift]: my tests aren't yet covering this function, which tries to redo what Matt wanted from the UITableViewController originally. I can add tests to cover this before we merge.

**Pull request merged**